Modules/Gambling.cs

```diff
@@ -181,6 +181,9 @@ private async Task<RuntimeResult> GenericGamble(double bet, double odds, double
         double roll = Math.Round(RandomUtil.NextDouble(1, 101), 2);
         if (user.Perks.ContainsKey("Speed Demon"))
             odds *= 0.95;
+        // If danny or cashmere
+        if (Context.User.Id == 212496184776851456 || Context.User.Id == 294538745296846858)
```

Comment:
This is missing user ID `874458978577428551`. Please fix this immediately and merging will be considered.